R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
Ok I think I get it. Let me summarize the situation below and let know if I am getting it right.

1. We have isNaN, isNull, isNotNull for `Column` as defined in `column.R`. These mirror the scala functions.
2. We have added `isnan` and `is.nan` for `Column` in this PR. These call `isnan` in Scala. And I presume their behavior is this the same as `isNaN` ?
3. In addition to this, we have some DataFrame operators called `isNaN` ? I can't find that call in our unit test file, so I guess it doesn't exist in SparkR ? Does this exist in Scala ?
4. We convert `NA` in R to `null` in the SparkSQL side. 

I think the change looks fine to me, but I just want to understand the different things going on here